libraries/anvill_passes/include/anvill/IndirectJumpPass.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <anvill/ABI.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/Pass.h>
+
+namespace anvill {
+namespace {
+bool isTargetInstrinsic(const llvm::CallInst *callinsn) {
```

Comment:
When putting things in anonymous namespaces, I like to also use `static` (even though redundant) to really clarify that this function / variable will end up local to the translation unit into which it is included.